{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -1,13 +1,17 @@
+# This is the Dockerfile that builds a fully functional image of your app
 {% include 'misc/header.py' %}
 ARG DEPENDENCIES_VERSION=latest
+# Get the previously built image(Docker.base) with pip dependencies and extend it
 FROM {{cookiecutter.project_shortname}}-base:${DEPENDENCIES_VERSION}
 
 COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
+# install the app
```

Comment:
I would add a further comment to the file, that this Dockerfile should be kept in sync with ``./scripts/bootstrap``